etna/records/templatetags/records_tags.py

```diff
@@ -19,7 +19,8 @@ def record_url(record: Record, is_editorial: bool = False) -> str:
     """
     if is_editorial and settings.FEATURE_RECORD_LINKS_GO_TO_DISCOVERY and record.iaid:
         return f"https://discovery.nationalarchives.gov.uk/details/r/{record.iaid}"
```

Comment:
Could we use the new formats you've put in `TNA_URLS` in DF-708? (Obviously 708 will need merging first 🙂)

Happy for this to be taken up in another ticket/PR though. Would be good for consistency 👍

Reply:
Thanks, this is done.